programs/mango-v4/src/state/mango_account_components.rs

```diff
@@ -736,12 +952,62 @@ mod tests {
         assert_eq!(pos.avg_entry_price_per_base_lot, 100_000.0);
         assert_eq!(pos.avg_entry_price(&market), 10_000.0);
         assert_eq!(pos.break_even_price(&market), 9_800.0);
-        assert_eq!(pos.realized_pnl_native, I80F48::from(20_000));
+        assert_eq!(pos.realized_trade_pnl_native, I80F48::from(20_000));
+    }
+
+    #[test]
+    fn test_perp_realized_settle_limit_no_reduction() {
```

Comment:
nit: maybe a dumb test, which, starts with 0 position, and/or records 0 value trades and 0 fees would also be nice